lib/SIL/Verifier/SILVerifier.cpp

```diff
@@ -1520,6 +1562,26 @@ class SILVerifier : public SILVerifierBase<SILVerifier> {
               "Scope of the debug variable should have the same parent function"
               " as that of instruction.");
 
+    llvm::StringRef UniqueName = VarNames.insert(varInfo->Name).first->getKey();
+    if (!varInfo->Loc)
+      varInfo->Loc = inst->getLoc();
+    if (!varInfo->Loc)
+      varInfo->Loc = SILLocation::invalid();
+    VarID Key(varInfo->Scope ? varInfo->Scope : debugScope,
+              UniqueName, *varInfo->Loc);
+    auto CachedVar = DebugVarTypes.find(Key);
```

Comment:
You can potentially replace all this with
`if (DebugVarTypes.insert({Key, DebugVarTy}).second)`
insert returns a bool that says whether the insert happened or not.